digitalocean/resource_digitalocean_monitor_alert.go

```diff
@@ -0,0 +1,291 @@
+					"v1/insights/droplet/public_inbound_bandwidth",
+					"v1/insights/droplet/private_outbound_bandwidth",
+					"v1/insights/droplet/private_inbound_bandwidth",
+				}, false),
+			},
+
+			"compare": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(godo.GreaterThan),
+					string(godo.LessThan),
+				}, false),
+				Description: "The comparison operator to use for value",
+			},
+
+			"description": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Description of the alert policy",
+			},
+
+			"enabled": {
+				Type:     schema.TypeBool,
+				Default:  true,
+				Required: true,
+			},
+
+			"value": {
+				Type:         schema.TypeFloat,
+				Required:     true,
+				ValidateFunc: validation.FloatAtLeast(0),
+			},
+
+			"tags": tagsSchema(),
+
+			"alerts": {
+				Type:        schema.TypeList,
+				Computed:    false,
+				Required:    false,
+				Description: "List with details how to notify about the alert. Support for Slack or email.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"slack": {
+							Type:     schema.TypeList,
+							Required: false,
+							Computed: false,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"channel": {
+										Type:         schema.TypeList,
+										Computed:     false,
+										Required:     true,
+										Description:  "The Slack channel to send alerts to",
+										ValidateFunc: validation.StringIsNotEmpty,
+									},
+									"url": {
+										Type:             schema.TypeString,
+										Computed:         false,
+										DiffSuppressFunc: CaseSensitive,
+										Description:      "The webhook URL for Slack",
+										ValidateFunc:     validation.StringIsNotEmpty,
+									},
+								},
+							},
+						},
+						"email": {
+							Type:         schema.TypeList,
+							Computed:     false,
+							Required:     false,
+							ValidateFunc: validation.ListOfUniqueStrings,
+						},
+					},
+				},
+			},
+
+			"entities": {
+				Type:        schema.TypeSet,
+				Required:    true,
+				MinItems:    1,
+				Description: "The droplets to apply the alert policy to",
+			},
+
+			"window": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"5m", "10m", "30m", "1h",
+				}, false),
+				MinItems: 1,
+			},
+		},
+	}
+}
+
+func resourceDigitalOceanMonitorAlertCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	client := meta.(*CombinedConfig).godoClient()
+
+	alertCreateRequest := &godo.AlertPolicyCreateRequest{
+		Type:        d.Get("type").(string),
+		Description: d.Get("description").(string),
+		Tags:        expandTags(d.Get("tags").(*schema.Set).List()),
+		Compare:     d.Get("compare").(godo.AlertPolicyComp),
+		Window:      d.Get("window").(string),
+		Value:       d.Get("value").(float32),
```

Comment:
```suggestion
		Value:       float32(d.Get("value").(float64)),
```

The native Terraform float is a `float64`, so it needs to get converted to `float32`